vllm/v1/worker/tpu_model_runner.py

```diff
@@ -1501,3 +1529,32 @@ def _get_padded_token_len(paddings: list[int], x: int) -> int:
     index = bisect.bisect_left(paddings, x)
     assert index < len(paddings)
     return paddings[index]
+
+
+def replace_set_lora(model):
+
+    def _tpu_set_lora(self,
+                      idx: int,
+                      lora_a: torch.Tensor,
+                      lora_b: torch.Tensor,
+                      embeddings_tensor: Optional[torch.Tensor],
+                      bias: Optional[torch.Tensor] = None):
+        # index = torch.tensor([idx], dtype=torch.int32, device="xla")
+        index = idx
```

Comment:
We need to set `VLLM_XLA_CHECK_RECOMPILATION=0` because there's recompilation during the lora setup.

And we'd better also add to TODO to resolve that.